systems/registry/org/pkg/server/org_test.go

```diff
@@ -50,7 +50,7 @@ func TestOrgServer_Add(t *testing.T) {
 	orgRepo.On("GetMemberCount", mock.Anything).Return(int64(1), int64(0), nil).Once()
```

Comment:
We need to add unit tests that cover most of the cases of all the new added code!

Reply:
true ..I will add those.

Reply:
You can open a seperate issue for that